python/date_module.py

```diff
@@ -0,0 +1,87 @@
+import mgp
+import pytz
+import datetime
+
+MINUTES_IN_HOUR = 60
+SECONDS_IN_MINUTE = 60
+MILLISECONDS_IN_SECOND = 1000
+HOURS_IN_DAY = 24
+UNIX_EPOCH = datetime.datetime(1970, 1, 1, 0, 0, 0)
+
+def getOffset(timezone, date):
```

Comment:
Are there any Typings from pytz library we can use here? So that we have suggestion which type is sent here

Reply:
Aha, typings yet. Yes sorry. So on Python, you have typings, https://docs.python.org/3/library/typing.html. Are there any available on `pytz` library which you can use in this case?